src/vs/workbench/contrib/output/browser/output.contribution.ts

```diff
@@ -368,9 +368,24 @@ class OutputContribution extends Disposable implements IWorkbenchContribution {
 					menu: {
 						id: MenuId.CommandPalette,
 					},
+					description: {
+						description: 'workbench.action.openLogFile',
+						args: [{
+							name: 'args',
+							schema: {
+								type: 'object',
+								properties: {
+									logFile: {
+										description: nls.localize('logFile', "The name of the log file to open, for example \"Extension Host\""),
```

Comment:
1. Why is `args` is an object? I would prefer to take single argument of type `string`. If there are more options we need in future, they can be passed as other arguments.
2. I would use log id instead of name. Because id is unique but name is translated and won't work in other languages.

Reply:
1. If it's a string that's fine, but I've found it difficult to discover how to use it as intellisense doesn't work for just string afaik.
2. How does a user get the ID? Do we need to show the id in the quick pick here where default is?
![image](https://github.com/microsoft/vscode/assets/2193314/dfc80faa-bd24-4be7-8c78-aff37b8fcd33)

Reply:
1. Then intellisense should be improved IMO
2. We could if necessary. For now, I would recommend users to check the following commands to get the ID
![image](https://github.com/microsoft/vscode/assets/10746682/1d9f6a92-f8d3-4a99-9e43-91dd54ad24b6)